samples/waveidl/idllexer/idl_re2c_lexer.hpp

```diff
@@ -116,7 +116,6 @@ template <typename IteratorT, typename PositionT>
 inline
 lexer<IteratorT, PositionT>::~lexer() 
 {
-    boost::wave::cpplexer::re2clex::aq_terminate(scanner.eol_offsets);
```

Comment:
I don't really remember why `aq_terminate` was in here. Is it not needed anymore?

Reply:
It is some kind of cleanup code that I don't understand, but what I do know is it already gets called from scanner's destructor :)

Reply:
Also the equivalent call is not present in cpplexer.

Reply:
Ok, thanks for fixing this!